packages/svelte/src/preprocessors.ts

```diff
@@ -48,6 +53,13 @@ export function componentTrackingPreprocessor(options?: ComponentTrackingInitOpt
       return { code: updatedCode, map: updatedSourceMap };
     },
   };
+
+  const sentryPreprocessor: SentryPreprocessorGroup = {
+    ...preprocessor,
+    id: FIRST_PASS_COMPONENT_TRACKING_PREPROC_ID,
```

Comment:
m: should we call this `sentry_id` so it doesn't collide with anything? More likely to be safer if Svelte chooses to use `id` in the future.